src/wallet/bdb.h

```diff
@@ -194,9 +194,8 @@ class BerkeleyCursor : public DatabaseCursor
     bool m_first{true};
 
 public:
-    // Constructor for cursor for all records
-    explicit BerkeleyCursor(BerkeleyDatabase& database, BerkeleyBatch* batch=nullptr) : BerkeleyCursor(database, batch, {}) {}
-    // Constructor for cursor for records matching the prefix
```

Comment:
In f76830e5:

This db cursor changes need to be in the first commit as it's not compiling currently.

Reply:
Oops, fixed.